pkg/os/console.go

```diff
@@ -0,0 +1,29 @@
+package os
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// PromptPasswords prompts the user for passwords with the given titles
+func PromptPasswords(passwordTitles []string) ([]string, error) {
+	reader := bufio.NewReader(os.Stdin)
+	passwords := make([]string, len(passwordTitles))
+
+	// iterate over password titles and prompt for each
+	for i, title := range passwordTitles {
+		fmt.Printf("%s Password: ", title)
+		password, err := reader.ReadString('\n')
+		if err != nil {
+			return nil, err
+		}
+
+		// trim delimiters
+		password = strings.TrimSuffix(password, "\n")
```

Comment:
I think we should use `strings.TrimSpace` for trimming both space and `\n`
https://play.golang.com/p/cI8trSutz26

Reply:
https://github.com/zeta-chain/node/pull/2673/commits/5d114881b9236c1b4cc7b0a4c1431832c2ad8988